int32 number_of_tags # The number of tags identified
int32[] tag_id       # The ids of the tags identified
float32[] tag_location_x    # The x-locations of the tags identified
float32[] tag_location_y    # The y-locations of the tags identified